internal/cli/atlas/streams/connection/connection.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code was autogenerated at 2023-07-13T05:55:59Z. Note: Manual updates are allowed, but may be overwritten.
+
+package connection
+
+import (
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "connection"
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: cli.GenerateAliases(use),
```

Comment:
this helper takes plural and camelCase and generates singular and kebab-case aliases, if you don't want `connections` to be the default then remove the generate alias

Reply:
Good point, I double checked the scope and it looks like we want "instance" and "connection", so I'll make this change.

Reply:
I think connections as the default is fine. I've made this change to pluralize instance as well. [3456422](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/3456422486683c26c0c3aac663e4b4fcae8f221b)